libtenzir/builtins/operators/http.cpp

```diff
@@ -831,15 +798,28 @@ struct from_http_args {
     return std::nullopt;
   }
 
-  auto make_headers() const -> std::unordered_map<std::string, std::string> {
+  auto make_headers() const
+    -> std::pair<std::unordered_map<std::string, std::string>,
+                 detail::stable_map<std::string, secret>> {
```

Comment:
Just stumbled over these two separate types. Probably can stay as-is, but this made me search for a reason and I couldn't fine one.